### PR TITLE
kostik/update-tables-xlsx/OYE-602

### PR DESCRIPTION
**Pull request by kostik700015**
That PR updates the aact_tables.xlsx file adding missing tables that we can export but not listed in aact_tables.xlsx and removing tables that are listed in aact_tables.xlsx but couldn't be exported. 

Added in the table:

ipd_information_types
mesh_headings
mesh_terms
reported_event_totals
retractions

Removed from the table: 

tagged_terms
analyzed_studies
cdek_organizations
cdek_synonyms
oncology_studies
mental_health_studies
cardiovascular_studies

### CODE REVIEW

File: public/documentation/aact_tables.xlsx
Comment (Javier-Jimenez-18) [inline public/documentation/aact_tables.xlsx:1]: I couldn't open the file. 

Is OYE-602 correct naming for AACT?

Comment (kostik700015) [reply public/documentation/aact_tables.xlsx:1]: Typo, it is AACT-602

Comment (kostik700015) [reply public/documentation/aact_tables.xlsx:1]: > I couldn't open the file.

I used Libre office. MS Excell also should work.

**Review by micronix — APPROVED** [review]
looks good

**Pull request merged**